multinet/migrations/util.py

```diff
@@ -0,0 +1,60 @@
+"""Utilities for migrations."""
+import inspect
+from datetime import datetime
+from arango.collection import StandardCollection
+from multinet.db import system_db
+from multinet.migrations.base import BaseMigration
+from multinet.migrations.types import Migration, MigrationDocument
+from types import ModuleType
+from typing import List, Generator
+
+
+# TODO: Add creation and return of `migrations` collection in _system db. This will
+# hold a document for each migration that exists. This can be used to check the
+# migrations that have been run, what needs to be run, etc.
+
+
+def is_migration(cls: type) -> bool:
+    """
+    Return true if the supplied class is an implemented Migration.
+
+    Only strict subclasses of the base Migration class are considered valid.
+    """
+    return issubclass(cls, BaseMigration) and cls != BaseMigration
+
+
+def get_migrations(modules: List[ModuleType]) -> List[Migration]:
+    """Return all migrations from the supplied list of modules."""
+
+    migrations: List[Migration] = []
+    for module in modules:
+        migrations.extend(
+            [
+                obj
+                for _, obj in inspect.getmembers(module, inspect.isclass)
+                if is_migration(obj)
+            ]
+        )
+
+    return migrations
+
+
+def migrations_collection() -> StandardCollection:
+    """Return the collection that stores previously applied migrations."""
+
+    if not system_db().has_collection("migrations"):
+        return system_db(False).create_collection("migrations")
+
+    return system_db(False).collection("migrations")
```

Comment:
Making the boolean argument usage more explicit will aid in readability:
```suggestion
        return system_db(readonly=False).create_collection("migrations")

    return system_db(readonly=False).collection("migrations")
```